command/agent.go

```diff
@@ -25,32 +29,29 @@ var Agent = cli.Command{
 		client := &http.Client{
 			Timeout: time.Second * 3,
 		}
-		req, err := http.NewRequest("GET", "http://localhost:8000/backups", nil)
-		if err != nil {
-			log.Fatal("Error reading request. ", err)
-		}
-		auth.AddAuthHeader(req, tokenResponse)
 
-		resp, err := client.Do(req)
-		if err != nil {
-			log.Fatal("Error reading response. ", err)
-		}
-		defer resp.Body.Close()
+		backups := map[string]actions.Backup{}
+		crons := map[string]*cron.Cron{}
 
-		body, err := ioutil.ReadAll(resp.Body)
-		if err != nil {
-			log.Fatal("Error reading body. ", err)
-		}
+		cr := cron.New()
+		cr.AddFunc("* * * * *", func() {
+			log.Println("Checking for changes to backups...")
+			newBackups := actions.ListBackups(client, tokenResponse, actions.BACKUP_TYPE_SCHEDULED)
 
```

Comment:
Here would be a good spot for some more logging. Seeing "Checking for changes to backups..." in the output with no other lines could disconcerting. I'd like to see the number of backups fetched, even if none were found or none of them are scheduled. Even a debug log line showing "Found 5 backups, all are already scheduled" would be very useful.